mlx_graphs/data/__init__.py

```diff
@@ -0,0 +1,6 @@
+# from .data import GraphData # noqa
+# from .batch import (GraphDataBatch, batch, unbatch) # noqa
+# from .collate import collate # noqa
+
+# TODO: when uncommenting, this generates a circular import error, need tovfix that
+# to import GraphData with from mlx_graphs.data import GraphData
```

Comment:
I don't see why that's happening. I just gave it a try and I don't have a circular import error when doing `from mlx_graphs.data import GraphData`.

Reply:
FWIW - all tests run well on my end

Reply:
Did you change the occurrences: `from mlx_graphs.data.data import GraphData` to `from mlx_graphs.data import GraphData`?

Reply:
I did, both in `test_data` and `test_batch`